osu.Game.Tests/Visual/Online/TestSceneChangelogOverlay.cs

```diff
@@ -155,6 +158,7 @@ public void ShowWithBuild()
             AddUntilStep(@"wait for streams", () => changelog.Streams?.Count > 0);
             AddAssert(@"correct build displayed", () => changelog.Current.Value.Version == "2018.712.0");
             AddAssert(@"correct stream selected", () => changelog.Header.Streams.Current.Value.Id == 5);
+            AddAssert(@"supporter promo showed", () => changelog.ChildrenOfType<ChangelogSupporterPromo>().First().Alpha == (isSupporter ? 0 : 1));
```

Comment:
These tests are failing because they're not waiting for the content to actually be loaded (they're async loaded). Needs a

```csharp
AddUntilStep(@"wait for content load", () => changelog.ChildrenOfType<ChangelogContent>().Any());
```

Reply:
Ah, thanks!